django_socio_grpc/services/servicer_proxy.py

```diff
@@ -264,32 +271,40 @@ def __getattr__(self, action):
 
     def process_exception(self, exc, request_container: GRPCRequestContainer):
         if isinstance(exc, GRPCException):
-            logger.error(
-                f"{type(exc).__name__} : {exc}",
-                exc_info=exc,
-            )
             request_container.context.abort(exc.status_code, exc.get_full_details())
-        elif isinstance(exc, grpc.RpcError) or request_container.context._state.aborted:
-            raise exc
         else:
-            logger.error(
-                f"{type(exc).__name__} : {exc}",
-                exc_info=exc,
-            )
             request_container.context.abort(grpc.StatusCode.UNKNOWN, str(exc))
 
-    async def async_process_exception(self, exc, request_container: GRPCRequestContainer):
+    async def async_process_exception(self, exc, context):
         if isinstance(exc, GRPCException):
-            logger.error(
-                f"{type(exc).__name__} : {exc}",
-                exc_info=exc,
-            )
-            await request_container.context.abort(exc.status_code, exc.get_full_details())
-        elif isinstance(exc, (grpc.RpcError, grpc.aio.AbortError)):
-            raise exc
+            await context.abort(exc.status_code, exc.get_full_details())
+        else:
+            await context.abort(grpc.StatusCode.UNKNOWN, str(exc))
+
+    def log_request_success(self, response, request_container):
```

Comment:
As per https://docs.djangoproject.com/en/4.2/ref/logging/#django-request the logging of success response is in fact not necessary